components/Withdraw.tsx

```diff
@@ -133,7 +133,7 @@ const Withdraw: React.FC<{}> = () => {
           await api.rpc.ethy.getEventProof(eventProofId)
         ).toJSON();
         if (versionedEventProof !== null) {
-          eventProof = versionedEventProof.EventProof;
+          eventProof = versionedEventProof.eventProof;
```

Comment:
I think the case is required to convert properly
```suggestion
          eventProof = versionedEventProof.EventProof;
```

Reply:
not sure why this has changed, I had to make this same change in [this PR](https://github.com/cennznet/bridge-ui/pull/43/commits/dad99e29535af2b9defe63dce4049b1425e476f8) in order for it to work - using the UI with `EventProof` returns eventProof as undefined

Reply:
I think that PR using cennznet/api 2.x.x version. in 1.5.x still lowercase. caused by polkadot/api changes in between the versions

Reply:
wrote this comment on the wrong PR several times 🤦‍♀️

Reply:
ahh yup forgot to reset yarn